__mocks__/keycloak-admin.ts

```diff
@@ -1,5 +1,5 @@
-import KcAdminClient from 'keycloak-admin';
+/* import KcAdminClient from 'keycloak-admin';
```

Comment:
Could we remove the commented out code instead? What is the purpose of this code being commented?